utils/setter.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+// SetValue assign value to the field
+// name of the struct dest.
+// Returns an error if name is not found or dest cannot
+// be addressed.
+func SetValue(name string, dest interface{}, value string) error {
```

Comment:
I'd prefer if the signature was clearer, like:

```
SetValue(structPtr interface{}, fieldName, value string)
```

Also, I'd move all the strconv.Unquote work into this func.

Reply:
Also, as a nit, I'd probably just make this func panic instead of returning an error. An error should only occur if there's a bad bug in our code, in which case panicking is fine. And your code will get much cleaner.

Reply:
I can't move the Unquote in this func as it is not needed for the `convert`.